### ISSUE TITLE
SSL: CERTIFICATE_VERIFY_FAILED

### ISSUE THREAD

**Issue by docdyhr** [satwikkansal/wtfpython#200]
It seems that the CLI version with '**pip install wtfpython -U**' no longer works! I got the following error when trying to install wtfpython: 
```
Fetching the latest version...
<urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1108)>
Uh oh, failed to check for the latest version, using the local version for now.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/bin/wtfpython", line 8, in <module>
    sys.exit(load_and_read())
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/wtf_python/main.py", line 31, in load_and_read
    render_doc()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/wtf_python/main.py", line 24, in render_doc
    with open(file_name, 'r', encoding='utf-8') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'content.md'
```

**Comment by satwikkansal** [comment]
Strange, I thought https://github.com/satwikkansal/wtfpython/commit/54883f5134896870436f716b849e935a23e23340 would have fixed the issue. Can you verify what version do you have installed

**Comment by docdyhr** [comment]
Still not workning.. with the same error message as above. python3 -m pip uninstall wtfpython:
Found existing installation: wtfpython 0.2.1

**Comment by nifadyev** [comment]
Hey @satwikkansal , shall this issue be closed due to #345 ?
